docs/www/configuration.md

```diff
@@ -466,6 +466,14 @@ pandaproxy_client:
   scram_password: ""
 
 rpk:
+  # Add optional flags to run rpk with specific parameters.
+  # The available start flags are found in: /src/v/config/configuration.cc
+  additional_start_flags:
```

Comment:
is this valid syntax? don't you need the dash? like so:

```
  additional_start_flags:    
  -  --overprovisioned    
  - --smp="2"
  - --memory="4G"
  - --default-log-level=info
```

Reply:
I just followed the format of the rest of the config.
@0x5d Can you confirm whether we need dashes?

Reply:
@bmansheim you can also run this on your laptop and verify no?

Reply:
Yeah, you need dashes and no trailing commas:

```yaml
  additional_start_flags:
    - "--overprovisioned"
    - "--smp=2"
    - "--memory=4G"
    - "--default-log-level=info"
```

Reply:
Yep. I've been sick though so I thought it would be quicker to get David's confirmation.